oxcaml/tests/backend/oxcaml_dwarf/dune.inc

```diff
@@ -0,0 +1,47 @@
+
+(rule
+ (enabled_if
+  (and
+   (= %{context_name} "main")
+   (= %{env:TEST_OXCAML_DWARF=false} "true")))
+ (targets test_basic_dwarf.exe)
+ (deps test_basic_dwarf.ml)
+ (action (run %{bin:ocamlopt.opt} %{deps} -g -gno-upstream-dwarf -bin-annot-cms -o test_basic_dwarf.exe)))
+
+(rule
+ (enabled_if
+  (and
+   (= %{context_name} "main")
+   (= %{env:TEST_OXCAML_DWARF=false} "true")
+   (<> %{env:OXCAML_LLDB=} "")))
+ (targets test_basic_dwarf.output.corrected)
+ (deps test_basic_dwarf.exe test_basic_dwarf.lldb filter.sh)
+ (action
+  (progn
+   (bash "sed -e 's/^(lldb) //' -e '/^[[:space:]]*$/d' test_basic_dwarf.lldb > test_basic_dwarf_clean.lldb")
+   (with-outputs-to test_basic_dwarf.output.corrected
+    (pipe-outputs
+     (run %{env:OXCAML_LLDB=lldb} -s test_basic_dwarf_clean.lldb ./test_basic_dwarf.exe)
```

Comment:
(Does not matter, but do we need the
fallback value for `env` here? The rule
appears to be executed only if we do
have a value.)

Reply:
(or maybe rather: if we do have a
fallback, maybe it should one such
that it is obvious something
unexpected is happening)